monai/losses/dice.py

```diff
@@ -671,6 +671,7 @@ def __init__(
 
         """
         super().__init__()
+        reduction = DiceCEReduction(reduction).value
```

Comment:
we prefer `monai.utils.look_up_option` in this case, it'll give better error message when there a typo:

```py
>>> look_up_option('eedge', NumpyPadMode)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/Documents/MONAI/monai/utils/module.py", line 120, in look_up_option
    raise ValueError(
ValueError: By 'eedge', did you mean 'edge'?
'eedge' is not a valid value.
Available options are {'constant', 'mean', 'linear_ramp', 'wrap', 'median', 'reflect', 'symmetric', 'minimum', 'maximum', 'edge', 'empty'}.

>>> look_up_option('edge', NumpyPadMode)
<NumpyPadMode.EDGE: 'edge'>
```